codespell_lib/data/dictionary.txt

```diff
@@ -25562,7 +25604,7 @@ wiew->view
 wigdet->widget
 wigdets->widgets
 wighed->weighed, wicked,
-wight->weight, white, right, wight,
+wight->weight, white, right, write, wight,
```

Comment:
I wonder if this should be disabled due to https://en.wikipedia.org/wiki/Isle_of_Wight

Reply:
i disagree because wight is a name (which can be diabled via the user using the `-L` flag). In my estimation this would most likely will be a typo

Reply:
We've already got `tim->time, Tim, disabled due to being a person's name`. Personally I think we should be consistent, it will still potentially flag up depending on your config. Otherwise we're back in that world of having valid words in our dictionary of typos because they're "rare", which rather depends on what you're doing.

Reply:
It's your call. `tim` is a lot more problematic that `wight`. but based on your above logic I'm ok to disable `wight`.

Reply:
I think that we can use some judgment here. At the 100% consistency end we'd have to search each "typo" to see if anyone anywhere has ever that name, and that doesn't make much sense. So it probably should be some sort of judgment call. In this case I think that the benefits of flagging the many errors that `wight` could represent outweight the costs of forcing an addition to `ignore_words` for people who want to use `wight` as a proper noun.